Applications/Resnet/PyTorch/recorder_v2.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+##
+# Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
```

Comment:
2023  Seungbaek Hong ?

Reply:
these files are copy of test/input_gen --> so i think there is no change

Reply:
I copied that python script temporarily. I will fix it to import original python script today. thanks!